data/model/oci/test/test_oci_tag.py

```
@@ -1013,3 +1014,95 @@ def test_delete_tags_for_manifest_skips_immutable(self, initialized_db):
 
         # v1.0 should still exist
         assert get_tag(repo.id, "v1.0") is not None
+
+
+class TestSetTagsImmutabilityForManifest:
+    def test_sets_all_alive_tags_immutable(self, initialized_db):
```

Ruff (0.14.13)

Unused method argument: `initialized_db`

(ARG002)